### ISSUE TITLE
enums working partially for WebStorm 2020.2.2

### ISSUE THREAD

**Issue by AMerkuri** [JetBrains/web-types#52]
If you have defined CSS declaration in your project for attribute then Webstorm will show you (ctrl + J) CSS specificity instead of providing autocomplete for web-types enums.

css
```css
foo[bar] { display: block }
```
html
```html
<foo bar=""></foo>
```
web-types.json
```json
...
  "contributions": {
    "html": {
      "elements": [
        {
          "name": "foo",
          "attributes": [
            {
              "name": "bar",
              "value": {
                "kind": "plain",
                "type": "enum"
              },
              "values": [
                {
                  "name": "baz"
                },
                {
                  "name": "qux"
                },
              ]
            },
          ]
...
```

**Comment by piotrtomiak** [comment]
@AMerkuri - thanks for the report. I've filed it in our tracking system - https://youtrack.jetbrains.com/issue/WEB-57066

**Comment by piotrtomiak** [comment]
The fix will be available in 2022.3.
